django/src/rdwatch/models/site_image.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.postgres.indexes import GistIndex
+from django.db import models
+from django.dispatch import receiver
+
+
+class SiteImage(models.Model):
+    siteeval = models.ForeignKey(
+        to='SiteEvaluation',
+        on_delete=models.CASCADE,
+        help_text='The site evaluation associated with this observation.',
+        db_index=True,
+    )
+    siteobs = models.ForeignKey(
+        to='SiteObservation',
+        on_delete=models.CASCADE,
+        help_text='The site Observation associated with this Image if it exists.',
+        null=True,
+        db_index=True,
+    )
```

Comment:
I think we can just have a single foreign key to `SiteObservation`, and if we need to access the `SiteEvaluation` we can do a join.

Reply:
Correct me if I'm wrong, but for some images a `siteObservation` may not exist?  It's why I made it nullable.  The movement of this to a separate table was to get images that may be outside of the list of siteObservations.

Reply:
Ah, right, I forgot there are some instances like that. This makes sense then